src/Components/StatBox/index.jsx

```
@@ -3,6 +3,7 @@ import Image from "../Image";
 import { useTheme } from "@mui/material/styles";
 import PropTypes from "prop-types";
 import useUtils from "../../Pages/Uptime/Monitors/Hooks/useUtils";
+import { lighten } from "@mui/material/styles";
```

ESLint

[error] 'lighten' is defined but never used.

(no-unused-vars)